### DISCUSSION TITLE
update firefox-addon

### DISCUSSION THREAD

**Discussion by rajarshikhatua** [Authenticator-Extension/Authenticator#1218] [General]
in [firefox's addon store](https://addons.mozilla.org/en-US/firefox/addon/auth-helper) the extention was updated over a year ago. it would be nice if dev can update the extension there for the ease of all users. thanks!

**Comment by mymindstorm** [comment]
Hi, we are working on it, just waiting to resolve a few issues before we update it over on Firefox.
